pom.xml

```diff
@@ -9,9 +9,9 @@
 		<relativePath />
 	</parent>
 
-	<groupId>org.retest</groupId>
+	<groupId>de.retest.services</groupId>
 	<artifactId>rebazer</artifactId>
-	<version>0.9.4-SNAPSHOT</version>
+	<version>0.10-SNAPSHOT</version>
```

Comment:
Intended to have no patch version anymore?

Reply:
Yes, this also comes from the sync.